VTEX - Logistics API.json

```diff
@@ -6364,575 +6243,714 @@
-                                                    },
-                                                    "example": {
-                                                        "zipCode": "22220070",
-                                                        "country": "BRA",
-                                                        "deliveryPointId": null,
-                                                        "point": null,
-                                                        "inStore": {
-                                                            "IsCheckedIn": false,
-                                                            "StoreId": "180082"
-                                                        }
                                                     }
                                                 },
                                                 "deliveryOnWeekends": {
-                                                    "type": "boolean"
+                                                    "type": "boolean",
+                                                    "description": "Defines if the delivery on weekends configuration is active (`true`) or inactive (`false`)."
+            
+                                                },
+                                                "weekendAndHolidays": {
+                                                    "type": "object",
+                                                    "description": "Delivery on weekends and holidays information.",
+                                                    "properties": {
+                                                        "saturday": {
+                                                            "type": "boolean",
+                                                            "description": "Defines if the delivery on Saturday configuration is active (`true`) or inactive (`false`)."
+                                                        },
+                                                        "sunday": {
+                                                            "type": "boolean",
+                                                            "description": "Defines if the delivery on Sunday configuration is active (`true`) or inactive (`false`)."
+                                                        },
+                                                        "holiday": {
+                                                            "type": "boolean",
+                                                            "description": "Defines if the delivery on [holiday](https://help.vtex.com/en/tutorial/registering-holidays--2ItOthSEAoyAmcwsuiO6Yk) configuration is active (`true`) or inactive (`false`)."
+                                                        }
+                                                    },
+                                                    "nullable": true
```

Comment:
o que acontece se for null?

Reply:
This is in the response, I assume it is because it was never configured.